libfwupdplugin/README.md

```diff
@@ -4,6 +4,8 @@ This library is only partially API and ABI stable. Keeping unused, unsafe and
 deprecated functions around forever is a maintenance burden and so symbols are
 removed when branching for new minor versions.
 
+Use `./contrib/migrate.py` to migrate up out-of-tree plugins to the new API.
```

Comment:
What about connecting this to pre commit?

Reply:
I did think about it, but it's a kinda dumb search-and-replace and I'm worried it's going to do bad things *tm* when run unconditionally. e.g. you still need to manually fix up the .map file for instance.

Reply:
I don't think running the script is in anyway helpful for an out-of-tree plugin; it would only be run when the source is changed (rather than treewide) and when just rebasing against master it won't do anything. Unless I'm missing the point somewhere.